docs/navigation-parameters-reference.md

```diff
@@ -111,4 +131,20 @@ The context switcher is a drop-down list available in the top navigation bar. It
   - **position** defines the action element position. Can be `top` or `bottom`. The default value is `top`. This parameter is optional.
   - **link** defines an absolute Link to a **node**. This parameter is optional.
   - **clickHandler** specifies a function and is executed on click and should return a boolean. If it returns `true`, **link** is opened afterwards.
-- **fallbackLabelResolver** specifies a function that is used to fetch the **label** for **options** that do not have a **label** defined. Additionally it fetches the dropdown label for non-existing **options**.
\ No newline at end of file
+- **fallbackLabelResolver** specifies a function that is used to fetch the **label** for **options** that do not have a **label** defined. Additionally it fetches the dropdown label for non-existing **options**.
+
+
+## Profile
+
+The profile section is a drop-down list available in the top navigation bar. Luigi configuration leverages the option to override the logout item content and/or add links to Luigi navigation nodes to the profile section. It just requires to add the `profile` property to the **navigation** object with any of these properties, all of them being optional:
```

Comment:
```suggestion
The profile section is a configurable drop-down list available in the top navigation bar. Within the configuration, you can override the logout item content or add links to Luigi navigation nodes. To do so, add the **profile** property to the **navigation** object using the following optional properties:
```

Reply:
You can actually do any of both or both. Why not keeping _and/or_?

Reply:
we can keep it :)